package.json

```diff
@@ -17,6 +17,7 @@
     "@elastic/ecs-pino-format": "^1.0.0",
     "@godaddy/terminus": "^4.6.0",
     "cors": "2.8.5",
+    "create-error": "^0.3.1",
```

Comment:
Use the `http-errors` module (already installed below).  I might have mislead you on which one to use, sorry about that.  They work the same way (e.g., both have `createError`), see https://www.npmjs.com/package/http-errors.

Reply:
Will do! Thanks for the heads up